debbuild/powerman-2.3.27-67-gc7cb796/src/httppower/httppower.c

```diff
@@ -0,0 +1,375 @@
+/*****************************************************************************
+ *  Copyright (C) 2007 The Regents of the University of California.
+ *  Produced at Lawrence Livermore National Laboratory (cf, DISCLAIMER).
+ *  Written by Jim Garlick <garlick@llnl.gov>
```

Comment:
bunch of accidental commits?  I see a bunch of `debbuild stuff`

Reply:
Crap!  don't know what happened there.

I just pushed an update that takes that stuff out and adds that directory to the gitignore.